source/_integrations/schedule.markdown

```diff
@@ -12,64 +12,52 @@ ha_domain: schedule
-have `default_config:` in your {% term "`configuration.yaml`" %}, it should already
-be there by default unless you removed it.
-
-If you removed `default_config:` from your configuration,
-you must add it back or, alternatively, `schedule:` to your
-`configuration.yaml` first, before you can create them via the UI.
+{% configuration_basic %}
+Name:
+  description: Friendly name of the schedule. It can be changed again later.
+Icon:
+  description: Icon to display in the frontend for this schedule.
+Schedule blocks:
+  description: >
+    Drag and drop with your mouse to select time boxes for each day of the week.
+    It is not possible to create overlapping time boxes on the same day.
+{% endconfiguration_basic %}
+
+After you have created schedule blocks, you can use a **single left click** on a block to open an edit view for this time block.
+
+{% configuration_basic  %}
+Start:
+  required: true
+  type: time
+  description: The start time to mark the schedule as active/on.
+End:
+  required: true
+  type: time
+  description: The end time to mark as inactive/off again.
+Additional data:
+  required: false
+  type: map
+  description: Additional data to add to the entity's attributes when this block is active.
+{% endconfiguration_basic %}
 
-Alternatively, a schedule can also be created and configured via YAML
-configuration. For example:
+The **Additional data** entry of each block should be a mapping of attribute names to values.
+For example, a block with the following data will show `brightness` and `color_temp` as {% term entity %} attributes when the block is active:
 
 ```yaml
-# Example configuration.yaml entry
-schedule:
-  thermostat_schedule:
-    name: "Thermostat schedule"
-    monday:
-      - from: "17:00:00"
-        to: "21:00:00"
-    tuesday:
-      - from: "17:00:00"
-        to: "21:00:00"
-    wednesday:
-      - from: "17:00:00"
-        to: "21:00:00"
-    thursday:
-      - from: "17:00:00"
-        to: "21:00:00"
-    friday:
-      - from: "17:00:00"
-        to: "23:00:00"
-    saturday:
-      - from: "07:00:00"
-        to: "10:00:00"
-      - from: "16:00:00"
-        to: "23:00:00"
-    sunday:
-      - from: "07:00:00"
-        to: "21:00:00"
+brightness: 100
+color_temp: 4000
 ```
 
-Defining the schedule in YAML also allows adding extra data to each block, which will
-appear as attributes on the schedule helper entity when that block is active. This can
-be used to easily build schedule-based automations.
+## YAML configuration
 
-The `data` key of each block should be a mapping of attribute names to values. In this example,
-the schedule helper entity will have "Brightness" and "Color temp" attributes when
-the blocks are active:
+Alternatively, this {% term integration %} can be configured and set up manually via YAML instead.
+To enable the Integration sensor in your installation, add the following to your {% term "`configuration.yaml`" %} file:
```

Comment:
I think something is missing here, or the goal of this sentence is pretty unclear.

Reply:
Deleted a little bit too much/too little.